.tool-versions

```diff
@@ -1 +1 @@
-ruby 2.7.1
+ruby 2.7.8
```

Comment:
This is the last version of ruby 2.7 that was released, able to install it via `ruby-build` on m2/m3 macs.